src/webgl.js

```diff
@@ -278,8 +274,8 @@ function init_conv1_1_program(gl) {
     out1 = vec4(0.0, 0.0, 0.0, 0.0);
     out2 = vec4(0.0, 0.0, 0.0, 0.0);
     out3 = vec4(0.0, 0.0, 0.0, 0.0);
```

Comment:
when all positions are 0 you can specify just once: `out1 = vec4(0.0)`. I think this is cleanup moreso than performacne.

Reply:
Updated!